### PR TITLE
STACK-884 : Renamed every tasks files to X_tasks.py

### PR DESCRIPTION
**Pull request by afrin-chakure**
## Closes
[STACK-884](https://a10networks.atlassian.net/browse/STACK-884)

### CODE REVIEW

File: a10_octavia/controller/worker/tasks/a10_compute_tasks.py [hunk 1]
```diff
@@ -26,13 +26,16 @@
 LOG = logging.getLogger(__name__)
```
Comment (omkartelee01) [inline a10_octavia/controller/worker/tasks/a10_compute_tasks.py]: Remove this file from this PR

**Comment by hthompson6** [comment]
Heads up, this will need more added to it after the other PR's get merged.

**Comment by hthompson6** [comment]
Please squash commit history down to just: "Renamed every tasks files to X_tasks.py"

**Comment by afrin-chakure** [comment]
Closing this PR because of unnecessary commit issues. 
Instead raised a new PR https://github.com/a10networks/a10-octavia/pull/55 regarding the same.

**Pull request closed without merging**